src/files.js

```diff
@@ -431,6 +431,31 @@ module.exports = (common) => {
             }
           })
       })
+
+      it.only('errors on unknown path', () => {
```

Comment:
You should remove the `.only` and also on line 443

Reply:
@richardschneider sorry, forgot to push.